usaspending_api/common/management/commands/matview_runner.py

```diff
@@ -0,0 +1,111 @@
+import asyncio
+import logging
+import psycopg2
+import subprocess
+
+from django.core.management.base import BaseCommand
+from pathlib import Path
+
+from usaspending_api.common.data_connectors.async_sql_query import async_run_creates
+from usaspending_api.common.helpers.timing_helpers import Timer
+from usaspending_api.common.matview_manager import (
+    DEFAULT_MATIVEW_DIR,
+    DROP_OLD_MATVIEWS,
+    MATERIALIZED_VIEWS,
+    MATVIEW_GENERATOR_FILE,
+    OVERLAY_VIEWS,
+)
+from usaspending_api.common.helpers.sql_helpers import get_database_dsn_string
+
+logger = logging.getLogger("console")
+
+
+class Command(BaseCommand):
+
+    help = "Create, Run, Verify Materialized View SQL"
+
+    def faux_init(self, args):
+        self.matviews = MATERIALIZED_VIEWS
+        if args["only"]:
+            self.matviews = {args["only"]: MATERIALIZED_VIEWS[args["only"]]}
+        self.matview_dir = args["temp_dir"]
+        self.no_cleanup = args["leave_sql"]
+
+    def add_arguments(self, parser):
+        parser.add_argument("--only", choices=list(MATERIALIZED_VIEWS.keys()))
+        parser.add_argument(
+            "--leave-sql",
+            action="store_true",
+            help="Leave the generated SQL files instead of cleaning them after script completion.",
+        )
+        parser.add_argument(
+            "--temp-dir",
+            type=Path,
+            help="Choose a non-default directory to store materialized view SQL files.",
+            default=DEFAULT_MATIVEW_DIR,
+        )
+
+    def handle(self, *args, **options):
+        """Overloaded Command Entrypoint"""
+        with Timer(__name__):
+            self.faux_init(options)
+            self.generate_matview_sql()
+            self.create_views()
+            if not self.no_cleanup:
+                self.cleanup()
+
+    def generate_matview_sql(self):
+        """Convert JSON definition files to SQL"""
+        if self.matview_dir.exists():
+            logger.warn("Clearing dir {}".format(self.matview_dir))
+            recursive_delete(self.matview_dir)
+        self.matview_dir.mkdir()
+
+        # NOTE TO SELF!!!!!!!
+        # DO NOT LEAVE hardcoded `python` in the command!!!!!!!
+        exec_str = "python {} --quiet --dest={}/ --batch_indexes=3".format(MATVIEW_GENERATOR_FILE, self.matview_dir)
```

Comment:
Originally, my goal was to use this function for operation jobs. There were issues I had running locally, it seems to be a macos issue not a python issue.
Since this isn't ready for operations and is for developers and other users I left it in. I should update the comment